pkg/client/create_builder_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"runtime"
```

Comment:
I would have expected at least one test in this file, given the nature of the changes. It seems to me that we should define the user experience when providing multi-arch buildpacks. Must they exist in a registry? Etc.

Reply:
Discussed with @jjbustamante, we'll add an acceptance test for multi-arch builder creation. Due to the nature of our test setup this will be simpler and easier.